src/main/client-side/extensions/merge-conflict-detector/merge-conflict-detector-extension.js

```diff
@@ -0,0 +1,87 @@
+import { ModalExtension } from '@atlassian/clientside-extensions';
+import 'regenerator-runtime/runtime'
+/**
+ * @clientside-extension
+ * @extension-point bitbucket.ui.pullrequest.overview.summary
+ */
+export default ModalExtension.factory((pluginApi, context) => {
+    return {
+        hidden: false,
+        label: 'Merge Conflict Detector',
+        onAction: modalApi => {
+            modalApi.onMount(container => {
+                var clean = true;
+                modalApi.setWidth('x-large');
+                modalApi.setTitle('Merge Conflict Detector');
+                modalApi.setActions([
+                    {
+                        text: 'Close',
+                        onClick: () => {
+                            var attributes = clean ? {iconBefore: 'app-access'} : {iconBefore: 'warning'};
```

Comment:
Can we default the modal link icon to a question mark of some kind?

Reply:
Yeah we can do it.  I ran into this -> https://jira.atlassian.com/browse/BSERV-12408

However, I found that if you nullify iconBefore before changing it, it works.